packages/vaadin-grid/src/vaadin-grid-helper.js

```diff
@@ -0,0 +1,12 @@
+export function updateColumnOrders(columns, scope, order) {
```

Comment:
Please addd license header in the beginning of the file.
Also, this function needs JSDoc annotation to explain what it does.